.github/workflows/ci.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
     strategy:
       matrix:
-        os: [ubuntu-22.04, macos-12, windows-2022]
+        os: [ubuntu-22.04, macos-12]
```

Comment:
@dmcgowan Why was the windows linter removed?

Reply:
Throwing the formatting error. If you have any ideas about it, we can re-enable